src/tests/tracing/eventpipe/Directory.Build.targets

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+
+  <Import Project="$([MSBuild]::GetPathOfFileAbove(Directory.Build.targets, $(MSBuildThisFileDirectory)..))" />
+
+  <!-- Hack to get NativeAOT assemblies into IlcReference
+       In CoreCLR tests, these assemblies get copied to CORE_ROOT, which NativeAOT doesn't use
```

Comment:
The comment makes it seem like all NativeAOT tests would hit this problem. Is this something that should be done at a higher level in the test infrastructure? What is it about these tests versus the ones we already run that make this necessary?

Reply:
It doesn't look like this impacts the wider test bed, at least the set of tests that are already enabled in categories like the loader and interop. The specific issue impacting the `eventpipe `tests are due to the [logging ](https://github.com/dotnet/runtime/blob/main/src/tests/tracing/eventpipe/common/Microsoft.Diagnostics.NETCore.Client/Microsoft.Diagnostics.NETCore.Client.csproj#L25)package reference bringing in the abstract logging package. I'll modify comments to reflect this.

Reply:
For me the overarching question is whether EventPipe testing really needs out-of-band assemblies - the tests we have in src/tests are more "runtime tests" than "libraries tests". If we need to test out-of-band assemblies, the more natural fit for that testing is in src/libraries - I sort of question why things that we don't even ship in the product are getting tested in src/tests to begin with.

This works for Mono/CoreCLR only because we dump all out-of-band assemblies into CORE_ROOT at some point during CORE_ROOT construction. It make CORE_ROOT a lot bigger than it needs to be.